product-service/src/get-products.ts

```diff
@@ -0,0 +1,49 @@
+import { connectDb } from './common/connect-db';
+import { res } from './common/res';
+import { Client } from 'pg';
+
+export const getProducts = async event => {
+  console.log('getProducts EVENT: ', event);
+
+  const client: Client = await connectDb();
+
+  if (!client) {
+    return res().sendInternal();
+  }
+
+  try {
+
+    await client.query(`
```

Comment:
I dont think that this is a good idea to create tables in getProducts method. Maybe it will be better to create another one or just to create sql file with steps for DB configuration

Reply:
Oh, I see you have that sql files